test_common/harness/errorHelpers.h

```diff
@@ -60,6 +60,11 @@ extern "C" {
 #define ct_assert_i(b, line)  ct_assert_ii(b, line)
 #define ct_assert_ii(b, line) int _compile_time_assertion_on_line_##line[b ? 1 : -1];
 
+#define test_fail(msg, ...)                                                    \
```

Comment:
Could this be superseded by `test_assert_error` in #912, which adds similar functionality?

Reply:
I have another PR to upload for a different issue which also uses this define. Would you mind if I left this for now and I'll go back and swap all instances for test_assert_error once the PRs are merged?

Reply:
Works for me!